webhook/handlers/patient_first_workflow.js

```diff
@@ -0,0 +1,79 @@
+const db = require('../../models')
+
+class PatientFirstController {
+
+  async bookConsulation(agent, id){
+      let fulfillmentText = "";
+      let user = await db.userWA.findOne({where: {wa_phone_number: id}});
+      let patients = await db.patientWA.findAll({where: { User_id : user.id }});
+      if(patients.length == 0){
+          fulfillmentText += ("There are no patients registered for you. Please register the patient(s) before booking the consultation.\n"
+              + "Would you like to register a patient now?\n")
+      }
+      else
+      {
+          fulfillmentText += "You have " + (patients.length).toString() + " patient" +
+          ((patients.length === 1) ? 's': '') + " registered currently:";
+          for(let i=0; i< patients.length; i++) {
+              fulfillmentText += "\nPatient " + (i+1).toString() + ": " + patients[i].first_name + " " + patients[i].last_name;
+          }
+          fulfillmentText += ("\nReply back the full name of the patient you wish to book a consult for or add a new patient.");
```

Comment:
Should change the last part so that user replies `yes` in case they want to add new patient: 
```
Reply back the full name of the patient you wish to book a consultation for. 
Or, if they are not present in the list, would you like to register them as a new patient?
```